R/derive_param_extreme_record.R

```diff
@@ -71,8 +71,8 @@
 #' The input dataset with the first or last observation of each by group
 #' added as new observations.
 #'
-#' @family der_prm_bds_findings
-#' @keywords der_prm_bds_findings
+#' @family superseded
+#' @keywords internal
```

Comment:
I think `@keywords superseded` should be used and a "Superseded" section should be added to the "Reference" page. Superseded functions could be used in submissions. If the health authority looks at the admiral webpage and can't find the function, it looks bad.

@bms63 , @manciniedoardo , do you agree?

Reply:
Yes, completely agree. Our discussion around "superseded" mentioned that we want to _allow users to still use the functions_, even if we'd _encourage them to use whatever new function came after them_.

Reply:
The help page still appears on the website and is easily found by searching the website. This will waste space on the already large Reference page. 

Regardless, please feel free to update as you'd like.

Reply:
I think a "superseded" section is a good call on the reference page.  

Hopefully, less deprecations will be occurring so will gain back some real estate.  I think we should also remove some of the other sections that users won't really care about

Reply:
I thought the testthat page mightve needed updates for messaging but I suppose superseding is just messaging without warning/errors?